src/main.js

```diff
@@ -28,10 +28,11 @@ window.addEventListener("load", onload);
 // FUNCTIONS:
 function findIdeas(event) {
   var foundIdeas = [];
-  var inputByUser = event.target.value
-  console.log (event.target.value)
+  var inputByUser = event.target.value.toLowerCase()
```

Comment:
How come toLowerCase?

Reply:
When we performed the search it was taking lowercase and uppercase into account, so we leveled the playing field by making everything to lowercase before our comparison.